ConsoleApplicationGoBoard/ConsoleFunctions.cpp

```diff
@@ -0,0 +1,87 @@
+#include "ConsoleFunctions.h"
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+void ConsoleFuntions::PrintBoard(GoBoard &goBoard)
+{
+	board CurrentBoard = goBoard.ReturnGoBoard();
```

Comment:
why is this an uppercase 'C' when you're using a lowercase 'g' for your GoBoard instance?

prefer to stick to one convention, so I would change it to "currentBoard"